modules/serverless-creating-integrationsink-aws-sns.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/event-sinks/serverless-integrationsink.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-creating-integrationsink-aws-sns_{context}"]
+= Creating an IntegrationSink for AWS SNS
+
+You can create an `IntegrationSink` API resource to publish CloudEvents from Knative Eventing to an Amazon Simple Notification Service (SNS) topic by applying a YAML manifest.
+
+.Prerequisites
+
+* You have created your AWS credentials and stored them in a Kubernetes Secret in the same namespace as the resource.
+* You have the OpenShift CLI (oc) installed and are logged in to the cluster.
+* You know the namespace where the `IntegrationSink` resource will be created.
+* A Knative broker or another event source exists to produce CloudEvents to be delivered to this sink.
+
+.Procedure
+
+. Save the following YAML manifest as `integration-sink-aws-sns.yaml`:
++
+[source,yaml]
+----
+apiVersion: sinks.knative.dev/v1alpha1
+kind: IntegrationSink
+metadata:
+  name: integration-sink-aws-sns
+  namespace: knative-samples
+spec:
+  aws:
+    sns:
+      arn: "my-topic"
```

Comment:
Since other examples use ARN syntax, I think it should be used here too - `arn:aws:sns:<region>:<account>:my-topic`

region used below is `eu-north-1`

Reply:
both works - just the topic name or the arn.  For simplicity we picked just `arn` (over `arnOrTopic (etc)Name`).

Reply:
If both work, then I will keep it as it is

Reply:
But you changed it, fine with me :)